benchmark/benchmark_test.go

```diff
@@ -6,8 +6,8 @@ import (
 	"testing"
 	"time"
 
+	testpb "google.golang.org/grpc/benchmark/grpc_testing"
 	"google.golang.org/grpc/benchmark/stats"
-	testpb "google.golang.org/grpc/interop/grpc_testing"
 )
 
 func run(b *testing.B, maxConcurrentCalls int, caller func(testpb.TestServiceClient)) {
```

Comment:
revoke my previous comment regarding renaming "run" since it seems runStream is not needed.